lms/djangoapps/certificates/models.py

```diff
@@ -574,3 +579,51 @@ def get_config(cls):
         instance = cls.current()
         json_data = json.loads(instance.configuration) if instance.enabled else {}
         return json_data
+
+
+class BadgeTracker(models.Model):
```

Comment:
I find the name `BadgeTracker` really confusing.  This model describes badges, not badge trackers, so it should be called `Badge`.  The reason you write something into the database always is that you want to keep track of it, but you still don't call the model keeping track of users `UserTracker`. :)